frontend/src/citizen-frontend/calendar/reservation-modal/TimeInputs.tsx

```diff
@@ -177,10 +175,11 @@ const ReadOnlyDay = React.memo(function ReadOnlyDay({
         </>
       )
     case 'holiday':
+      // doesn't go here, bug?
       return (
-        <FixedSpaceRow fullWidth>
+        <FixedSpaceRow fullWidth alignItems="center">
           <LeftCell>{label}</LeftCell>
-          <MiddleCell textOnly>{i18n.calendar.holiday}</MiddleCell>
+          <MiddleCell>{i18n.calendar.holiday}</MiddleCell>
```

Comment:
I added a holiday to db but still it didn't go to this branch (was case undefined) and didn't say pyhäpäivä on the UI

Reply:
I'll fix in the next PR